### PR TITLE
Move community meeting to a monthly cadence

### PR DESCRIPTION
**Pull request by castrojo**
fixes https://github.com/kubernetes/steering/issues/150

### CODE REVIEW

**Comment by tpepper** [comment]
This lgtm and has been discussed/decided at the SIG ContribEx level (eg: https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit#bookmark=id.iebp4xpxqpk3), but I would like to
/hold
at least for a few days in case there's any broader discussion following the k-dev announcement of intent at:
https://groups.google.com/d/msg/kubernetes-dev/JU9XIx6UUWk/L8BQNYjYAAAJ

**Comment by idvoretskyi** [comment]
@tpepper no broader discussion has happened on k-dev during the last 14 days.

I assume was may merge this.

**Comment by parispittman** [comment]
we need to reconcile the governance requirements before this can be merged

"Report activity in the weekly community meeting at least once every quarter"

cc @kubernetes/steering-committee

**Comment by parispittman** [comment]
documenting idea from slack convos:
make kubecon intros a requirement and then post videos and decks somewhereTBD to meet the quarterly ask

**Comment by nikhita** [comment]
Lgtm :+1:

**Comment by parispittman** [comment]
we need to incorporate the where people can find those updates more here or how we as a community are going to collect the updates. the previous statement about the community meeting was firm and straightforward and we are losing that here.

File: committee-steering/governance/sig-governance.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ In order to standardize Special Interest Group efforts, create maximum transpare
 - Meet regularly, at least for 30 minutes every 3 weeks, except November and December
 - Keep up-to-date meeting notes, linked from the SIG's page in the community repo
 - Record meetings and make them publicly available on a YouTube playlist
-- Report activity in the weekly community meeting at least once every quarter
+- Report activity with the community at least once a quarter via any of the following means:
```
Comment (cblecker) [inline committee-steering/governance/sig-governance.md]: If we are providing multiple avenues for updates, should contribex help collect those together and send out a mailing each quarter with them? Would this give us any benefit? Will contribex continue to track these updates, and raise to steering any SIGs that are not in compliance?

Comment (castrojo) [reply committee-steering/governance/sig-governance.md]: I pushed a PR to make it more explicit that k-dev should be the final destination for a status update.

**Comment by derekwaynecarr** [comment]
this lgtm as well.

/lgtm

**Pull request merged**